model/SyncService.php

```diff
@@ -493,4 +506,18 @@ public function getSynchronizer($type)
         return $this->synchronizers[$type];
     }
 
+    /**
+     * @param array $params
+     * @return int
+     * @throws \core_kernel_persistence_Exception
+     */
+    private function getSyncId(array $params)
```

Comment:
this function rather is about to create createSynchronisation than get an Id.
IMHO name `createSynchronisation` is better reflects its intention.